tidy3d/plugins/autograd/utilities.py

```diff
@@ -77,6 +77,23 @@ def make_kernel(kernel_type: KernelType, size: Iterable[int], normalize: bool =
     return kernel
 
 
+def get_kernel_size_px(radius=None, dl=None):
+    if radius is None or dl is None:
+        raise ValueError("Either 'size_px' or both 'radius' and 'dl' must be provided.")
```

Comment:
If we are restrictive in the kwargs we accept earlier on, probably this will never need to be checked?

Reply:
I though this here would be the common denominator for all functions that use `get_kernel_size_px`, so it makes to check this here? Otherwise we'd need to add this check to every functions that handles both `radius` and `dl`?